src/openrct2/ride/Vehicle.cpp

```diff
@@ -9838,7 +9838,7 @@ void Vehicle::Serialise(DataSerialiser& stream)
     stream << var_C0;
     stream << speed;
     stream << powered_acceleration;
-    stream << dodgems_collision_direction;
+    stream << CollisionDetectionDirection;
```

Comment:
As the majority of users will be the timer switch all the things not dodgem related to use the CollisionDetectionTimer